### PR TITLE
Generating .NET SDK for Paloaltonetworks

### PR DESCRIPTION
**Pull request by Vibhuti-Sharma-Microsoft**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/ci.yml
Comment (ArcturusZhang) [inline sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/ci.yml:1]: Your PR might be generated from a previous version of the project template, therefore the ci file content here is kind of out-of-date.
Could please update it according to the example here? https://github.com/Azure/azure-sdk-for-net/blob/76ff14813aeb4248856c1df9e7032c45d2234f90/sdk/compute/ci.mgmt.yml#L2
Including: 
- file name should be `ci.mgmt.yml`
- file content, I suggest you override the whole file with the example and change the directory name and namespace back to yours.

File: sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure PaloAltoNetworks management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/README.md:3]: Add description of your service here

File: sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/src/autorest.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+azure-arm: true
+csharp: true
+library-name: PaloAltoNetworks
+namespace: Azure.ResourceManager.PaloAltoNetworks
```
Comment (ArthurMa1978) [inline sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/src/autorest.md:9]: Has this namespace been reviewed?

File: sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/src/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("PaloAltoNetworks")]
```
Comment (ArthurMa1978) [inline sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/src/Properties/AssemblyInfo.cs:6]: Should be `PaloAltoNetworks.Cloudngfw`

File: sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/ci.yml
Comment (Vibhuti-Sharma-Microsoft) [reply sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/ci.yml:1]: done

File: sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/README.md (hunk 1 above)
Comment (Vibhuti-Sharma-Microsoft) [reply sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/README.md]: done

File: sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
[the remark this answers is not in the document]
Comment (Vibhuti-Sharma-Microsoft) [reply sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/CHANGELOG.md]: done

File: sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/src/autorest.md (hunk 2 above)
Comment (Vibhuti-Sharma-Microsoft) [reply sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/src/autorest.md]: Namespace review is complete now - https://github.com/Azure/azure-sdk-pr/issues/892
The approved namespace is for .NET is "Azure.ResourceManager.PaloAltoNetworks.Ngfw".
I have regererated the code to reflect this change.

File: sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/src/Properties/AssemblyInfo.cs (hunk 3 above)
Comment (Vibhuti-Sharma-Microsoft) [reply sdk/paloaltonetworks/Azure.ResourceManager.PaloAltoNetworks/src/Properties/AssemblyInfo.cs]: Namespace review is complete now - https://github.com/Azure/azure-sdk-pr/issues/892
The approved namespace is for .NET is "Azure.ResourceManager.PaloAltoNetworks.Ngfw".
I have regererated the code to reflect this change.

**Comment by ArthurMa1978** [comment]
Namespace review is done https://github.com/Azure/azure-sdk-pr/issues/892

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**